### PR TITLE
Adding companyDetails in the json

### PR DESCRIPTION
**Pull request by sampankumar**
Adding companyDetails in the json
__JIRA Ticket Number__
IDVA5-772
### Type of change

* [ x] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by hemerem-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/test/java/uk/gov/companieshouse/acsp/service/FilingServiceTest.java [hunk 1]
```diff
@@ -224,9 +288,65 @@ void tesGenerateAcspApplicationFilingWithBlankAddresses() throws Exception {
...
+        setACSPDataDtoWithCompanyDetailsButNoCompanyNumberAndName();
+        when(acspService.getAcsp(any(), any())).thenReturn(Optional.of(acspDataDto));
+        when(transactionService.getTransaction(PASS_THROUGH_HEADER, TRANSACTION_ID)).thenReturn(transaction);
+
+        var response = filingsService.generateAcspApplicationFiling(ACSP_ID, TRANSACTION_ID, PASS_THROUGH_HEADER);
+        Assertions.assertNull(((ACSP) response.getData().get("data")).getCompanyName());
+        Assertions.assertNull(((ACSP) response.getData().get("data")).getCompanyNumber());
+    }
+
+
```
Comment (sardhani-ch) [inline src/test/java/uk/gov/companieshouse/acsp/service/FilingServiceTest.java:349]: Can you remove commented lines from the tests.

Comment (sampankumar) [reply src/test/java/uk/gov/companieshouse/acsp/service/FilingServiceTest.java:349]: I wil have to add these back when I bring the payment bits back

**Pull request merged**